### ISSUE TITLE
[📚] Documentation Issue with remote config

### ISSUE THREAD

**Issue by SuperBesse** [invertase/react-native-firebase#4263]
## Documentation Feedback

In the current documentation, the method to get value from a remote key is illustrated by :

```
The API also provides a getAll method to read all parameters at once rather than by key:

const parameters = remoteConfig().getAll();

Object.entries(parameters).forEach(([key, parameter]) => {
  console.log('Key: ', key);
  console.log('Value: ', parameter.value);
});

```
https://rnfirebase.io/remote-config/usage

But actually, this method **value** doesn't exist. I have to use private var **_value** to get the correct value and not undefined.
In fact, i don't know if documentation is wrong or it's an issue from implementation ?

---

- 👉 Check out [`React Native Firebase`](https://twitter.com/rnfirebase) and [`Invertase`](https://twitter.com/invertaseio) on Twitter for updates on the library.

**Comment by mikehardy** [comment]
Hi there - thanks for pointing that out. There is something ...not great about the docs website right now.

1.  I'm not sure what's going on but the examples from the actual type file aren't showing. So you can't see in the associated reference API site the wonderful example: https://github.com/invertase/react-native-firebase/blob/9097f8e794f161f0c46bf1d54ae1e43386fbbb2c/packages/remote-config/lib/index.d.ts#L430-L447

2. That doc is out of date - https://github.com/invertase/react-native-firebase/blame/master/docs/remote-config/usage/index.md#L121-L128 - and should be updated to reflect the above - if you want to post a quick PR we can merge it - GitHub makes docs PRs really easy, every docs page has an edit button at top right and then in github just clicking the little edit pen icon gets an easy web-based flow to make a change

**Comment by SuperBesse** [comment]
Ok thanks for fast answer. I test this part and I will propose a PR to update documentation.

**Comment by mikehardy** [comment]
Already proposed the PR and merged - awesome! #4268
